elf2cfetbl_version.h

```diff
@@ -39,14 +41,19 @@
 #define ELF2CFETBL_MINOR_VERSION 1  /*!< @brief Minor version number */
 #define ELF2CFETBL_REVISION      99 /*!< @brief Revision version number. Value of 99 indicates a development version.*/
```

Comment:
REVISION should be 0 instead of MISSION_REV being 00. Can use to set version string to development vs release.